alembic/versions/20231020_7fceb9488bc6_drop_cachedfeeds.py

```diff
@@ -0,0 +1,92 @@
+"""Drop cachedfeeds
+
+Revision ID: 7fceb9488bc6
+Revises: 0739d5558dda
+Create Date: 2023-10-20 10:55:49.709820+00:00
+
+"""
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "7fceb9488bc6"
+down_revision = "0739d5558dda"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic ###
+    op.drop_index("ix_cachedfeeds_lane_id", table_name="cachedfeeds")
+    op.drop_index("ix_cachedfeeds_library_id", table_name="cachedfeeds")
+    op.drop_index(
+        "ix_cachedfeeds_library_id_lane_id_type_facets_pagination",
+        table_name="cachedfeeds",
+    )
+    op.drop_index("ix_cachedfeeds_timestamp", table_name="cachedfeeds")
+    op.drop_index("ix_cachedfeeds_work_id", table_name="cachedfeeds")
+    op.drop_table("cachedfeeds")
+    op.drop_column("works", "simple_opds_entry")
```

Comment:
We should be able to remove the `simple_opds_entry` field from the editions table as well. https://github.com/ThePalaceProject/circulation/blob/cd8ef179b0ec277151816522ebef7b34d3f2e403/core/model/edition.py#L141 It seems unused after these changes (possibly before as well?)

Reply:
Yes, I think before as well. Removing.